network/datadog_checks/network/network.py

```diff
@@ -433,6 +433,29 @@ def _check_linux(self, instance):
                     self._submit_netmetric(nstat_metrics_names[k][met], self._parse_value(netstat_data[k][met]),
                                            tags=custom_tags)
 
+        proc_conntrack_path = "{}/net/nf_conntrack".format(proc_location)
```

Comment:
These two blocks are very alike. Maybe we should write a parametrized function to avoid copy/past

Reply:
You mean the reading of the file? One counts the lines in the file and the other picks the number that is in it which is a quite different operation.

Reply:
Hum, my bad, you are right 😄